tools/manifest/manifest.py

```diff
@@ -475,7 +475,7 @@ def from_json(cls, tests_root, obj, types=None):
             # present when doing an incremental manifest update.
             # See https://github.com/web-platform-tests/rfcs/pull/27 for background.
             #
-            # TODO(MANIFESTv7): remove this condition
+            # TODO(MANIFESTv8): remove this condition
```

Comment:
You can just delete this branch, because we don't need to worry about older manifest versions.